README.md

```diff
@@ -293,165 +317,78 @@ export default MessageList = () => {
 
-Configure an instance of the [Auth service][] and pass it to the `Security` component.
+#### Example
 
 ```jsx
-// src/App.js
-
-import React, { Component } from 'react';
-import { Router, Route } from 'react-router-dom';
-import { AuthService, Security, LoginCallback, SecureRoute } from '@okta/okta-react';
-import Home from './Home';
-import Protected from './Protected';
+import { useHistory } from 'react-router-dom';
+import { OktaAuth } from '@okta/okta-auth-js';
 
-const authService = new AuthService({
+const oktaAuth = new OktaAuth({
   issuer: 'https://{yourOktaDomain}.com/oauth2/default',
   clientId: '{clientId}',
-  redirectUri: window.location.origin + '/implicit/callback',
+  redirectUri: 'window.location.origin' + '/login/callback'
 });
 
-class App extends Component {
-  render() {
-    return (
-      <Router>
-        <Security authService={authService} >
-          <Route path='/' exact={true} component={Home}/>
-          <SecureRoute path='/protected' component={Protected}/>
-          <Route path='/implicit/callback' component={LoginCallback} />
-        </Security>
-      </Router>
-    );
-  }
-}
+export default App = () => {
+  const history = useHistory();
 
-export default App;
+  const customAuthHandler = (oktaAuth) => {
+    // Redirect to the /login page that has a CustomLoginComponent
+    // This example is specific to React-Router
+    history.push('/login');
+  };
+
+  return (
+    <Security
+      oktaAuth={oktaAuth}
+      onAuthRequired={customAuthHandler}
+    >
+      <Route path='/login' component={CustomLoginComponent}>
+      {/* some routes here */}
+    </Security>
+  );
+};
 ```
 
 #### PKCE Example
 
-Assuming you have configured your application to allow the `Authorization code` grant type, simply pass `pkce=true` to the `Security` component. This will configure the [Auth service][] to perform PKCE flow for both login and token refresh.  
-
-```jsx
-
-class App extends Component {
-  render() {
-    return (
-      <Router>
-        <Security issuer='https://{yourOktaDomain}.com/oauth2/default'
-                  clientId='{clientId}'
-                  pkce={true}
-                  redirectUri={window.location.origin + '/implicit/callback'}>
-          <Router path='/login' component={CustomLoginComponent}>
-          {/* some routes here */}
-        </Security>
-      </Router>
-    );
-  }
-}
-```
+Assuming you have configured your application to allow the `Authorization code` grant type, you can implement the PKCE flow with the following steps:
 
-You may also configure an instance of the [Auth service][] directly and pass it to the Security component.
+- Initialize [oktaAuth](Okta Auth SDK) instance with `pkce=true` and pass it to the `Security` component.
```

Comment:
pkce is the default now, we should rephrase this.

Reply:
how about remove this sample? as initial `pkce` or not actually is the responsibility for `auth-js` instead of `okta-react`